andela/titan-dev-ah-frontend
# #164215462 create single article view

*What does this PR do?*

- create the view article page using HTML and CSS

*Description of Task to be completed?*

- Create the HTML and CSS page
- Make it responsive for both mobile and desktop

*How should this be manually tested?*

- Fetch origin
- Check out the branch ft-create-single-article-view-164215462
- Open the following file pages/article.html in chrome or firefox
- And use the inspector to view the page with different devices.

*Screenshot Desktop*

<img width="1318" alt="Screen Shot 2019-03-15 at 08 54 09" src="https://user-images.githubusercontent.com/18573157/54413939-32d67b80-4700-11e9-9ce5-6efe4787ca03.png">

* Screenshot Mobile *

<img width="596" alt="Screen Shot 2019-03-15 at 08 54 41" src="https://user-images.githubusercontent.com/18573157/54413963-3ff36a80-4700-11e9-8ac4-15e10c12a9fe.png">

<img width="454" alt="Screen Shot 2019-03-15 at 08 55 01" src="https://user-images.githubusercontent.com/18573157/54413972-441f8800-4700-11e9-81ba-5c5cb3412b5b.png">

*Any background context you want to provide?*

None

*What are the relevant pivotal tracker stories?*

• #164215462
*Questions:
None
